helm/values.yaml

```diff
@@ -401,4 +401,25 @@ docling:
     port: 15001
     type: ClusterIP
   imagePullPolicy: IfNotPresent
-  image: "ghcr.io/docling-project/docling-serve-cpu:v0.16.0"
\ No newline at end of file
+  image: "ghcr.io/docling-project/docling-serve-cpu:v0.16.0"
+
+## MCP Context Forge configuration
+forge:
+  image: ghcr.io/i-am-bee/mcp-context-forge:0.5.0
+  service:
+    type: ClusterIP
+    port: 4444
+  config:
+    CACHE_TYPE: "memory"
+    HOST: 0.0.0.0
+    PORT: 4444
+  secret:
+    AUTH_REQUIRED: "false"
+    DATABASE_URL: "sqlite:///data/forge.db"
+  volumeMounts:
+    - name: forge-storage
+      mountPath: /data
+  volumes:
+      - name: forge-storage
+        persistentVolumeClaim:
+          claimName: data-postgresql-0 # reuse postgres pvc
```

Comment:
what does this do? we can't mount the same postgres data to multiple services

Reply:
Resolved by standalone pvc